main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+)
+
+var (
+	airtableApiKey = os.Getenv("AIRTABLE_API_KEY")
+	airtableBaseId = os.Getenv("AIRTABLE_BASE_ID")
+)
+
+func healthHandler(w http.ResponseWriter, _ *http.Request) {
+	w.Write([]byte("PONG"))
```

Comment:
You can use, `io.WriteString(w, "PONG")` here.

https://golang.org/pkg/io/#WriteString